internal/service/address.go

```diff
@@ -0,0 +1,27 @@
+package service
+
+import (
+	"context"
+	"errors"
+
+	"github.com/thirdfort/thirdfort-go-code-review/internal/models"
+)
+
+func FindAddress(tx_id string) (models.Address, error) {
+	address, err := MyService.DataStore.FindAddress(context.TODO(), tx_id)
+	return *address, err
+}
+
+func UpdateAddress(transactionId string, newAddr models.Address) (*models.Address, error) {
+
+	address, _ := FindAddress(transactionId)
```

Comment:
FindAddress returns an error. Discarding the error with "_" is not ideal. I suggest getting the error returned and handling it